bindings/lni_nodejs/index.d.ts

```
@@ -3,6 +3,267 @@
+  nodeId: string
+  force: boolean
+}
+export interface UpdateChannelRequest {
+  channelId: string
+  nodeId: string
+  forwardingFeeBaseMsat: number
+  maxDustHtlcExposureFromFeeRateMultiplier: number
+}
+export interface CloseChannelResponse {
+  
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)